aldryn_search/helpers.py

```diff
@@ -1,4 +1,6 @@
 # -*- coding: utf-8 -*-
+from cms.plugin_rendering import PluginContext
+import django
```

Comment:
actually to keep consistency with the whole app, the cms should be after django since django is a lower level package but just a comment, no need to change.